src/native/external/libunwind-version.txt

```diff
@@ -1,10 +1,6 @@
-v1.6.2
-https://github.com/libunwind/libunwind/commit/b3ca1b59a795a617877c01fe5d299ab7a07ff29d
+main@e07b43c02d5cf1ea060c018fdf2e2ad34b7c7d80
+https://github.com/libunwind/libunwind/commit/e07b43c02d5cf1ea060c018fdf2e2ad34b7c7d80
 
 Reapply changes from https://github.com/dotnet/runtime/commit/1b5719c2e3dde393531eaeb5b5cde05dabeef5b8
-Apply https://github.com/libunwind/libunwind/pull/317
 Apply https://github.com/libunwind/libunwind/pull/333
 Apply https://github.com/libunwind/libunwind/pull/342
```

Comment:
I was trying to be conservative and include only the merged in changes. The PR 353 is an obvious bug fix though, so I agree that it would make sense to apply that one too. 
The other two are related to walking over a signal frame and we never do that, we unwind that differently in PAL_VirtualUnwind.